code/modules/buildmode/submodes/offer.dm

```diff
@@ -0,0 +1,29 @@
+// Speeds up the offering process and optionally allows the admin to set up playtime requirement and "Show role" only once.
+// Default setting is 20H like the default recommendation for offering from drop down menu.
+/datum/buildmode_mode/offer
+	key = "offer"
+	var/hours = 20
+	var/hide_role
+
+/datum/buildmode_mode/offer/show_help(mob/user)
+	to_chat(user, "<span class='notice'>***********************************************************</span>")
+	to_chat(user, "<span class='notice'>Left click to offer a mob</span>")
+	to_chat(user, "<span class='notice'>Right click to change amount of playtime a player needs to be able to sign up and whether to display their special role</span>")
+	to_chat(user, "<span class='notice'>***********************************************************</span>")
+
+/datum/buildmode_mode/offer/change_settings(mob/user)
+	hours = input(user, "Playtime required", "Input", 20) as num|null
+	if(alert("Do you want to show the mob's special role?", null, "Yes", "No") == "Yes")
```

Comment:
Since these aren't Super Crucial Admin Tools, you can probably use the tgui variants of these (tgui_alert/tgui_input_num)

Reply:
I'm not sure, I think the tgui ones prob dont make a lot of difference and just prob dont hurt eyes thats about it

Reply:
They don't yeah, in due time all of these in all buildmode variants should probably be changed